packages/renderer/client/motion-canvas.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="@motion-canvas/core/project" />
```

Comment:
What is this for?

Reply:
this imports type definitions we need. The compiler throws errors otherwise